ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -19,39 +16,49 @@ import com.daml.metrics.MetricsReporter
 import com.daml.platform.apiserver.SeedService.Seeding
 import com.daml.platform.configuration.Readers._
 import com.daml.platform.configuration.{CommandConfiguration, IndexConfiguration}
+import com.daml.platform.services.time.TimeProviderType
 import com.daml.platform.usermanagement.UserManagementConfig
 import com.daml.ports.Port
 import io.netty.handler.ssl.ClientAuth
 import scopt.OptionParser
 
+import java.io.File
+import java.nio.file.Path
+import java.time.Duration
+import java.util.UUID
+import java.util.concurrent.TimeUnit
 import scala.concurrent.duration.FiniteDuration
 
 final case class Config[Extra](
-    mode: Mode,
-    ledgerId: String,
-    commandConfig: CommandConfiguration,
-    tlsConfig: Option[TlsConfiguration],
-    participants: Seq[ParticipantConfig],
-    maxInboundMessageSize: Int,
-    configurationLoadTimeout: Duration,
-    maxDeduplicationDuration: Option[Duration],
-    eventsPageSize: Int,
-    eventsProcessingParallelism: Int,
-    acsIdPageSize: Int,
-    acsIdFetchingParallelism: Int,
+    allowedLanguageVersions: VersionRange[LanguageVersion],
+    authService: AuthService,
     acsContractFetchingParallelism: Int,
     acsGlobalParallelism: Int,
+    acsIdFetchingParallelism: Int,
+    acsIdPageSize: Int,
     acsIdQueueLimit: Int,
-    stateValueCache: caching.WeightedCache.Configuration,
-    lfValueTranslationEventCache: caching.SizedCache.Configuration,
+    configurationLoadTimeout: Duration,
+    commandConfig: CommandConfiguration,
+    enableInMemoryFanOutForLedgerApi: Boolean,
+    enableSelfServiceErrorCodes: Boolean,
```

Comment:
You have a race on that parameter with @kamil-da's PR

Reply:
Someone will have to rebase. I'm fine if it's going to be me